common.yaml

```diff
@@ -63,13 +63,6 @@ postprocess:
      #!/usr/bin/env bash
      set -xeo pipefail
 
-     # Disable PasswordAuthentication in SSH
-     sed -i "s|^PasswordAuthentication yes$|PasswordAuthentication no|g" /etc/ssh/sshd_config
-     # Disable root login because don't do that.
-     sed -i "s|^PermitRootLogin yes$|PermitRootLogin no|g" /etc/ssh/sshd_config
-     # Enable ClientAliveInterval and set to 180 per https://bugzilla.redhat.com/show_bug.cgi?id=1701050
-     sed -i "s|^#ClientAliveInterval 0$|ClientAliveInterval 180|g" /etc/ssh/sshd_config
-
```

Comment:
Did these sed statements stop working because the formatting in `/etc/ssh/sshd_config` changed?

Reply:
See the `^` in the sed lines

Reply:
The last one likely worked but I've just moved it for consistency.